wgengine/packet/packet.go

```diff
@@ -37,11 +40,53 @@ func (p IPProto) String() string {
 		return "UDP"
 	case TCP:
 		return "TCP"
+	case IPv6:
+		return "IPv6"
 	default:
 		return "Junk"
 	}
 }
 
+// These are the minimal sizes of the respective headers.
+// It is therefore not appropriate to use them during decoding.
+// They are used in packet assembly, where we always have minimal headers.
+const (
+	IPHeaderSize   = 20
```

Comment:
do these need to be exported?

Reply:
These, on the other hand, aren't currently used outside and probably don't need to be exported. I'll change that.